legoshop.py

```diff
@@ -271,15 +271,44 @@ def browser_info(self):
         print("* Browser wanted URL: {url}".format(url=self.shop_url))
         print("* Browser current URL: {url}".format(url=self.browser.current_url))
 
+    def _process_age_gate(self):
+        """
+        Continue Passed Age Gate
+        """
+        print("* Continue Passed Age Gate")
+        try:
+            self.wait \
+                .until(EC.element_to_be_clickable((By.XPATH, "//button[@data-test = 'age-gate-grown-up-cta']")))\
+                .click()
+        except NoSuchElementException:
+            print("!!! Something's wrong with the age gate button")
```

Comment:
Would it make sense to suggest a solution? (e.g. "Verify that ... foo ... bar.")